docs/launchpad/release-channels.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_position: 3
+---
+
+# Release Channels
+
+Due to the intricate nature of managing indexing operations for multiple blockchains and their associated dependencies, the Launchpad project is a complex system with numerous interdependencies. 
+
+For a reminder of the various components within Launchpad and their intricate connections, we recommend revisiting our [Intro](intro.md).
+
+This guide offers a comprehensive walkthrough, outlining the steps, automated and manual, required to introduce a new application, ie. Erigon, into the 'launchpad-charts' repository as a **canary** release and ultimately transitioning it to a **stable** state within its designated 'launchpad-namespace,' such as Ethereum. 
+
+The diagram below provides a visual representation illustrating the interdependence and impact of various components and workflows.
+
+![Release Channels Flow](/img/launchpad-release-channels.svg)
+
+# From new version to `launchpad-namespaces` stable
+
+Below you can find a more comprehensive breakdown of the process, divided into automated workflows within launchpad-charts and launchpad-namespaces, as well as manual operator steps. This process guides the transition of a new application version from the initial launchpad-charts canary release to its eventual stability within the corresponding launchpad-namespace. For this walkthrough we will use Erigon as an example.
+
+**launchpad-charts** 
+  - Bot watches [Erigon tags](https://github.com/ledgerwatch/erigon/tags) and as a new version is published it opens a PR into [`launchpad-charts/charts/erigon` ](https://github.com/graphops/launchpad-charts/pull/133)
+  - The new PR triggers a workflow that publishes a new [`pre-release`](https://github.com/graphops/launchpad-charts/releases/tag/erigon-0.8.1-canary.1) into the repo.
+  - Another workflow runs and adds the newly released `canary` chart to the `canary` Helm repo index
+
+**launchpad-namespaces** 
+  - Bot sees new chart release and pushes update branch and opens a [new PR to namespaces](https://github.com/graphops/launchpad-namespaces/pull/38)
+  - Bot runs again, and auto-merges the PR to create a tag, triggers workflow to create a release
```

Comment:
```suggestion
  - Bot runs again, auto-merges the PR and creates a tag
```

Reply:
I decided to not trigger creating a pre-release for namespaces given that, contrary to the charts, we do not need releases for "software distribution". The only purpose would be release notes, which aren't working all that great because of the open git-cliff bug. As such, thought we could skip those and just do the tag for canaries?
It's easy to add that trigger to the workflow, if that's the decision (and then add back that step here).

Reply:
this makes sense